compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -668,13 +760,51 @@ class Objects(using Context @constructorOnly):
    */
   def call(value: Value, meth: Symbol, args: List[ArgInfo], receiver: Type, superType: Type, needResolve: Boolean = true): Contextual[Value] = log("call " + meth.show + ", this = " + value.show + ", args = " + args.map(_.value.show), printer, (_: Value).show) {
     value.filterClass(meth.owner) match
-    case Cold =>
-      report.warning("Using cold alias. " + Trace.show, Trace.position)
+    case TopWidenedValue =>
+      report.warning("Value is unknown to the checker due to widening. " + Trace.show, Trace.position)
       Bottom
+    case UnknownValue =>
+      reportWarningForUnknownValue("Using unknown value. " + Trace.show, Trace.position)
+
+    case Package(packageModuleClass) =>
+      if meth.equals(defn.throwMethod) then
+        Bottom
+      // calls on packages are unexpected. However the typer might mistakenly
+      // set the receiver to be a package instead of package object.
+      // See packageObjectStringInterpolator.scala
+      else if !meth.owner.denot.isPackageObject then
```

Comment:
nitpick: the comment alignment looks a little strange. If the comment is for the else branch, maybe it's better to put it inside the branch.